ld.so.conf

```diff
@@ -0,0 +1,5 @@
+# We just make any GL32 extension have higher priority
```

Comment:
This file isn't needed [anymore](https://github.com/flathub/flathub/pull/2897#discussion_r822753402).

Reply:
ah ok, so the inline will write a file with the content. got it